ssm/transitions.py

```diff
@@ -222,6 +230,8 @@ def log_transition_matrices(self, data, input, mask, tag):
     def m_step(self, expectations, datas, inputs, masks, tags, **kwargs):
         Transitions.m_step(self, expectations, datas, inputs, masks, tags, **kwargs)
 
+    def hessian_expected_log_trans_prob(self, data, input, mask, tag, expected_joints):
```

Comment:
You don't need to explicitly call the parent class's method.  Here, I would actually return zeros since we know the transition potentials are indep of x.